core/src/components/modal/gestures/sheet.ts

```diff
@@ -42,6 +43,27 @@ export const createSheetGesture = (
   const backdropAnimation = animation.childAnimations.find(ani => ani.id === 'backdropAnimation');
   const maxBreakpoint = breakpoints[breakpoints.length - 1];
 
+  /* tslint:disable-next-line */
```

Comment:
Why the tslint disables? (Here and on L105)

Reply:
The linter errors out that the check is always true, but that isn't the case in SSR. We use this approach in almost all cases where we are checking if the document or window is available.

Reply:
Ah, gotcha 👍 I wasn't getting an error on my end after removing the disable comment, so I wasn't sure. I'm seeing it in other places though, like you said.